contracts/strategies/interfaces/liquity/IBorrowerOperations.sol

```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity >=0.6.0 <0.9.0;
```

Comment:
Let's use 0.8.6 for all solidity pragmas.

Reply:
ok, will update in next commit